cmake/modules/falcosecurity-libs.cmake

```diff
@@ -27,8 +27,8 @@ else()
   # In case you want to test against another falcosecurity/libs version (or branch, or commit) just pass the variable -
   # ie., `cmake -DFALCOSECURITY_LIBS_VERSION=dev ..`
   if(NOT FALCOSECURITY_LIBS_VERSION)
-    set(FALCOSECURITY_LIBS_VERSION "0.7.0")
-    set(FALCOSECURITY_LIBS_CHECKSUM "SHA256=3adc1620c0e830554a54cdd486158dc2c0c40552e113785b70fbbc99edb7d96f")
+    set(FALCOSECURITY_LIBS_VERSION "111135405708f69efa391a5140706e273917e28d")
```

Comment:
We need to also bump the driver version, otherwise driver loader integration tests fail because driver's API_VERSION was bumped.
See: https://app.circleci.com/pipelines/github/falcosecurity/falco/2891/workflows/dc8be5be-1bd9-46fe-a677-ff70ce6e24de/jobs/25514